src/Processor.js

```diff
@@ -57,9 +57,16 @@ class Processor {
         if (this.options.senderFnFactory) {
             this.senderFnFactory = this.options.senderFnFactory;
         } else {
+            const senderLogger = {
```

Comment:
Tohle by mel resit handler "onSenderError" - nemelo by to byt nutne. Pokud to nefunguje, je to bug.